src/coreclr/build.cmd

```diff
@@ -128,6 +128,8 @@ if /i "%1" == "-x86"                 (set __BuildArchX86=1&set processedArgs=!pr
 if /i "%1" == "-arm"                 (set __BuildArchArm=1&set processedArgs=!processedArgs! %1&shift&goto Arg_Loop)
 if /i "%1" == "-arm64"               (set __BuildArchArm64=1&set processedArgs=!processedArgs! %1&shift&goto Arg_Loop)
 
+if /i "%1" == "-linux"               (set __BuildOS=Linux&set processedArgs=!processedArgs! %1&shift&goto Arg_Loop)
```

Comment:
This option may imply general support.  Maybe a special purpose flag would be better.